core/tests/widgets/test_imageview.py

```diff
@@ -176,3 +176,21 @@ def test_rehint_empty_image(params):
     assert width == 0
     assert height == 0
     assert aspect_ratio is None
+
+def test_pil_support():
+    from PIL import Image as PIL_Image
+    pil_img = PIL_Image.open("resources/toga.png")
+    toga_img = PIL_Image.open("resources/toga.png")
+
+    imageview = toga.ImageView(pil_img)
+    assert type(imageview.image) == toga.Image, "Internal conversion from PIL_Image.Image to toga.Image is faulty"
+    assert (imageview.image.width, imageview.image.height) == (toga_img.width,toga_img.height) == pil_img.size, "PIL support for imageview is faulty"
```

Comment:
As in the Image test, we know the reference image, so we can test it's properties directly.